.github/workflows/validate-cg-manifest.sh

```diff
@@ -82,7 +83,7 @@ do
 
   # Pull the current registration from the cgmanifest file. Every registration should have a url, so if we don't find one
   # that implies the registration is missing.
-  manifesturl=$(jq --raw-output ".Registrations[].component.other | select(.name==\"$name\" and .version==\"$version\") | .downloadUrl" cgmanifest.json)
+  manifesturl=$(jq --raw-output ".Registrations[].component.other | select(.name==\"$name\" and (.version==\"$version\" or .version==\"$version_no_tilde\")) | .downloadUrl" cgmanifest.json)
```

Comment:
Would these changes now allow to remove some packages from the `ignore_list` at the top?

Reply:
Seems not, I don't see any intersection between the ignore list and the list of packages with a tilde in the version.